apps/www/registry/default/ui/alert-dialog.tsx

```diff
@@ -65,7 +65,7 @@ const AlertDialogFooter = ({
 }: React.HTMLAttributes<HTMLDivElement>) => (
   <div
     className={cn(
-      "flex flex-col-reverse sm:flex-row sm:justify-end sm:space-x-2",
+      "flex flex-col-reverse sm:flex-row sm:justify-end sm:space-x-2 rtl:space-x-reverse",
```

Comment:
Why not `flex flex-col-reverse sm:flex-row sm:justify-end sm:gap-2`?

Reply:
I didnt want to change the original styling alot. Just adding rtl:space-x-reverse at the end  is the most minimal, i think.